cmd/validate_image.go

```diff
@@ -220,6 +230,7 @@ Use an inline EnterpriseContractPolicy spec
 		"write output to a file. Use empty string for stdout, default behavior")
 	cmd.Flags().BoolVarP(&data.strict, "strict", "s", data.strict,
 		"return non-zero status on non-successful validation")
+	cmd.Flags().BoolVarP(&data.summary, "summary", "c", data.summary, "print condensed output of conftest")
```

Comment:
I think mentioning `conftest` here is not very helpful to the user. Maybe, something like `condense violation and warning messages`?

Reply:
I agree. updated.